src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /**
+     * @return the feedback to user
+     */
```

Comment:
consider removing auto-gen  header comments if it adds no value. Generally accessors don't need header comments as their names are self-explanatory.